### PR TITLE
Quick lang switch

### PR DESCRIPTION
**Pull request by trasher**

### CODE REVIEW

**Comment by trasher** [comment]
lang_glpi2

File: inc/switchlang.class.php [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+ You should have received a copy of the GNU General Public License
+ along with GLPI. If not, see <http://www.gnu.org/licenses/>.
+ --------------------------------------------------------------------------
+ */
+
+/** @file
+* @brief
+*/
+// And Marco Gaiarin for ldap features
```
Comment (AnaelMobilia) [inline inc/switchlang.class.php]: Copy/Paste glitch ?

**Comment by trasher** [comment]
I've remove the new class and added the method in the user class.

**Comment by orthagh** [comment]
front/ one could also be removed, no ?
I think about a new case in front/user.form.php

**Comment by trasher** [comment]
Done.

**Comment by wawax** [comment]
Hi trasher.
It doesn't work on simplified interface ^^

**Comment by trasher** [comment]
As far as I can see, I do not have any language switch on simplified interface; I'll check if I broke something or if it was "by design"

**Comment by wawax** [comment]
Hi trasher,

In simplified interface, the dropdown is in the user preferences.
I don't know why there was no link to change the language on the central page, but it seems to me that could be a good idea.
What do you think ?

**Comment by trasher** [comment]
Yes, I do not see why it should not be available to all user; since this is already possible from their preferences page :)

**Pull request merged**